builder/yandex/config.go

```diff
@@ -62,6 +62,9 @@ type Config struct {
 	// Key/value pair labels to
 	// apply to the created image.
 	ImageLabels map[string]string `mapstructure:"image_labels" required:"false"`
+	// Minimum size of the disk that will be created from built image, specified in gigabytes.
+	// Should be more or equal to `disk_size_gb`.
+	ImageMinDiskSizeGb int `mapstructure:"image_min_disk_size_gb" required:"false"`
```

Comment:
I'm curious here, what happens someone sets:

```json
"disk_size_gb": 42,
"image_min_disk_size_gb": 84,
```

This means the disk_size will be 42 at startup but then expand to 84 after installs ?

Reply:
Not certainly in that way. This new parameter is about what size the Disk will be when it is created from the Image that built by Packer.
So with 
```
"disk_size_gb": 42,
"image_min_disk_size_gb": 84,
```
when your create new Disk from Image, then you should specify Disk size equal or greater than `minDiskSize` property value of Image.

Also you may check this [doc](https://cloud.yandex.com/docs/compute/api-ref/Disk/create).

Reply:
Many thanks for the informations !!